stl/inc/format

```diff
@@ -902,7 +914,7 @@ private:
     _NODISCARD static constexpr int _Utf8_code_units_in_next_character(
         const char* const _First, const char* const _Last) noexcept {
         char32_t _Ch;
-        const char* _Next = _Decode_utf(_First, _Last, _Ch);
+        const char* const _Next = _Decode_utf(_First, _Last, _Ch);

```

Comment:
```suggestion
        const auto _Next = _Decode_utf(_First, _Last, _Ch);
```

I don't have a strong opinion, but I think this and line 982 should use the same convention here.

Reply:
Excellent catch, agreed. Local consistency is an important convention.

Reply:
yeah I don't care either, happy to adopt this suggestion.